common/config/bity.ts

```diff
@@ -11,7 +11,8 @@ const BTCMax = 3;
 // value = percent higher/lower than 0.01 BTC worth
 const buffers = {
   ETH: 0.1,
-  REP: 0.2
+  REP: 0.2,
+  XMR: 0.3
```

Comment:
A function lower in this file references `keyof buffers` and was upset when `XMR` didn't exist in this hash. I'm open to better typing suggestions.